tests/test_process.py

```diff
@@ -218,26 +211,22 @@ def test_running_calcjob_output_widget(generate_calc_job_node):
         }
     )
 
-    # Test the widget can be instantiated with a process
-    RunningCalcJobOutputWidget(calculation=process)
+    widget = RunningCalcJobOutputWidget()
```

Comment:
The `RunninngCalcJobOutputWidget` initializer does not have `calculation` as its input. This code was likely copy-pasted. This API is a bit unconsistent, since many other widgets allow passing the traitlet via constructor. But that's for another PR.